### ISSUE TITLE
TYPO3 9.5 compatibility

### ISSUE THREAD

**Issue by SvenJuergens** [pschriner/dm_develogerlog#7]
hi,

do you plan a release for TYPO3 9.5.x ?

Best Regards,
Sven

**Comment by pschriner** [comment]
Hi Sven,

I'm already working on it (branch pre-merge).

[Due to a change in what is accessible from the TYPO3 Bootstrap object](https://docs.typo3.org/typo3cms/extensions/core/Changelog/9.2/Important-83869-RemovedRequestTypeSpecificCodeInBootstrap.html) I'm still struggling with how I can get the request-id (I don't like my current reflection based version).

Since last week I've got an idea that it might work via a PSR-15 middleware, but that is still more of a guess (I have almost no clue how I can access the containers request.id field).

TL;DR: definitely planed in the near future.

**Comment by SvenJuergens** [comment]
hi patrick,

sounds good :)

maybe the Talk from Susi Moog about PSR15 can help? 
https://www.youtube.com/watch?v=lefgKJSWqx0&t=18116s

Best Regards
Sven
